src/main/groovy/com/blackducksoftware/integration/hub/docker/HubClient.groovy

```diff
@@ -109,63 +108,11 @@ class HubClient {
         hubServerConfigBuilder.proxyUsername = hubProxyUsername
         hubServerConfigBuilder.proxyPassword = hubProxyPassword
 
-        hubServerConfigBuilder
-    }
-
-
-    File retrieveHttpsCertificate(){
-        File certificateFile = new File('certificate.txt')
-        URL url = new URL(hubUrl)
-        String certificate = ''
-        def standardOut = new StringBuilder()
-        def standardError = new StringBuilder()
-        String command = "keytool -printcert -rfc -sslserver ${url.getHost()}"
-        Process proc = command.execute()
-        proc.consumeProcessOutput(standardOut, standardError)
-        proc.waitForOrKill(commandTimeout)
-        certificate = standardOut.toString()
-        logger.info("Retrieving the certificate from ${url.getHost()}")
-        logger.debug(certificate)
-        def error = standardError.toString()
-        if(StringUtils.isNotBlank(error)){
-            logger.warn(standardError.toString())
-        }
-        logger.debug("Exit code ${proc.exitValue()}")
-        if(proc.exitValue() != 0){
-            throw new HubIntegrationException("Failed to retrieve the certificate from ${hubUrl}")
+        if(autoImportCert == null){
```

Comment:
This makes sure that we do the import unless the explicitly set it to false, is that what we want? Or do we only want to do it if they set it to true?